src/main.rs

```diff
@@ -44,11 +47,13 @@ async fn rocket() -> Rocket<Build> {
 
     let client = setup_http_client();
     let cache = create_service_cache().await;
+    let mainnet_cache = create_service_cache_mainnet().await;
```

Comment:
As this is a quick fix I think this is ok. I would have preferred a more generic way. E.g. a pool manager, that has a mapping of chain id to pool (or service cache how we call it here). All the pools would be instantiated when the app starts, e.g. as defined in an env var that contains a json blob with the expected info :P